OCCommunicationLib/OCCommunicationLibTests/OCCommunicationLibTests.m

```diff
@@ -2161,6 +2161,119 @@ - (void) testGetFeaturesSupportedByServer {
 }
 
 ///-----------------------------------
+/// @name testGetFeaturesSupportedByServerForVersion
+///-----------------------------------
+
+- (void) privateTestGetFeaturesSupportedByServerForVersion:(NSString *)version expectedResults:(NSArray *)expectedResults {
+    OCServerFeatures *supportedFeatures = [_sharedOCCommunication getFeaturesSupportedByServerForVersion:version ];
+    
+    if (!supportedFeatures) {
+        XCTFail(@"Error reading server features for version %@, got NIL", version);
+    }
+    
+    if (supportedFeatures.hasShareSupport != [[expectedResults objectAtIndex:0] boolValue]) {
```

Comment:
You can use also XCTAssertEqualObjects(expected, result, @"If not equal show this error");